content/faq/security/general.md

```diff
@@ -20,25 +20,25 @@ Passwords aren't required to be periodically reset. NIST no longer recommends pa
 
 Docker Hub’s global setting for system lockout is after 10 failed sign in attempts in a period of 5 minutes, and the lockout duration is 5 minutes. The same global policy applies to authenticated Docker Desktop users and Docker Scout, both of which use Docker Hub for authentication.
 
-### Do you support physical MFA with YubiKeys? 
+### Do you support physical Multi-factor authentication (MFA) with YubiKeys? 
```

Comment:
Don't expand abbreviations in headings https://docs.docker.com/contribute/style/grammar/#acronyms-and-initialisms